.github/actions/nodejs/action.yml

```diff
@@ -43,16 +43,20 @@ runs:
         key: npm-cache__nodejs-${{ inputs.node-version }}__${{ hashFiles('**/package-lock.json') }}
         restore-keys: npm-cache-hash__
 
+    - shell: bash
+      run: |
+        echo "Show environment Node.js, npm ..."
+        node -v
+        npm -v
+
+    - shell: bash
+      run: |
+        echo "Provide global variables ..."
+        echo "NPM_CONFIG_CACHE=~/.npm" >> $GITHUB_ENV
+        echo "NODE_OPTIONS=--max-old-space-size=12288" >> $GITHUB_ENV
+        echo "CYPRESS_CACHE_FOLDER=node_modules/cache-cypress" >> $GITHUB_ENV
```

Comment:
nit: 
why that syntax `echo "key=value" >> $GITHUB_ENV`
instead of field `env: key: value` as before ?
I don't know reason, but if it's possible, I think second one is more readable